app/services/deploy_service.rb

```diff
@@ -3,6 +3,8 @@
 class DeployService
   attr_accessor :deploy_strategy
 
+  MERGE_PRIOR_WARNING = 1.hour
```

Comment:
Is it possible to set the auto deploy to happen after less than 1 hour?

Reply:
Technically, yes. Maybe this should just be a default, with an override supported in `arguments`? That would be pretty easy to add.

Reply:
(And what should that argument be named? I wasn't happy with this name.)

Reply:
not sure, this one is hard - maybe `NOTIFY_BEFORE_DEPLOY_TIME`